lib/Doctrine/ORM/UnitOfWork.php

```diff
@@ -804,6 +814,7 @@ public function computeChangeSet(ClassMetadata $class, $entity)
                     }
 
                     if ($orgValue !== null && $assoc['orphanRemoval']) {
+                        /** @psalm-var object $orgValue */
```

Comment:
We usually use `assert` instead of `@var`.

```suggestion
                        assert(is_object($orgValue));
```